src/styles/main.scss

```diff
@@ -0,0 +1,80 @@
+@mixin diameter ($name) {
+  min-width: $name;
+  height: $name;
+}
+
+$border-color: #2b2a2a;
+
+body,
+ul,
+li {
+  margin: 0;
+}
+
+.clock {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+
+  &__clockface {
+    @include diameter(300px);
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 0;
+    list-style-type: none;
+    border: 7px solid $border-color;
+    border-radius: 50%;
+    background-color: #e1f2fc;
```

Comment:
Pay attention to grouping your styles - https://codeguide.academy/html-css.html#css-order